modules/ROOT/pages/actors-index.adoc

```diff
@@ -1,21 +1,35 @@
 = Actors 
-:description: blabla
+:description: Actors are indirections between the processes and entities of the organization. They are used to define the candidates who can perform a given human task.
+
+Actors are indirections between the processes and entities of the organization. They are used to define the candidates who can perform a given human task. +
```

Comment:
```suggestion
{description}
```

Reply:
If you say this is how it should be said, so be it.